examples/docker-compose-modrinth-modpack.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+  mc:
+    image: itzg/minecraft-server
+    tty: true
+    stdin_open: true
+    ports:
+      - "25565:25565"
+    environment:
+      EULA: "TRUE"
+      TYPE: QUILT
```

Comment:
...and unfortunately Quilt won't be supported by the new mechanism at first since I also need to add support for that in mc-image-helper. I wanted to get the overall feature out ASAP...since as I can tell by your great contribution, people are very excited for this 😀

Reply:
I think many people are eager and willing to support open source projects like Modrinth. 😁